tests/daemon/mock-site-id-provider.c

```diff
@@ -23,19 +23,19 @@
 #include "emer-site-id-provider.h"
 #include <glib.h>
 
-static GVariant *
-emer_read_location_label (void)
+static void
+emer_read_location_label (GVariantBuilder *builder)
 {
-  g_auto (GVariantBuilder) builder = G_VARIANT_BUILDER_INIT (G_VARIANT_TYPE_ARRAY);
-
-  g_variant_builder_add (&builder, "{ss}", "id", "myid");
-  g_variant_builder_add (&builder, "{ss}", "country", "Earth");
-
-  return g_variant_builder_end (&builder);
+  g_variant_builder_add (builder, "{ss}", "id", "myid");
+  g_variant_builder_add (builder, "{ss}", "country", "Earth");
 }
 
 GVariant *
 emer_site_id_provider_get_id (void)
 {
-  return emer_read_location_label();
+  g_auto (GVariantBuilder) builder = G_VARIANT_BUILDER_INIT (G_VARIANT_TYPE("a{ss}"));
+
+  emer_read_location_label(&builder);
+
+  return g_variant_builder_end (&builder);
```

Comment:
Same style points as the real function here.

(You could inline `emer_read_location_label` here.)

Reply:
ok